clang-tools-extra/clang-tidy/tool/run-clang-tidy.py

```diff
@@ -198,9 +198,11 @@ def find_binary(arg: str, name: str, build_path: str) -> str:
 def apply_fixes(
     args: argparse.Namespace, clang_apply_replacements_binary: str, tmpdir: str
 ) -> None:
-    """Calls clang-apply-fixes on a given directory."""
+    """Calls clang-apply-replacements on a given directory."""
     invocation = [clang_apply_replacements_binary]
-    invocation.append("-ignore-insert-conflict")
```

Comment:
Actually this patch is now changing the default behavior of the tool, which is to ignore conflicts. Users updating the tool will get the opposite behavior without having done anything differently. How big of a deal is this?

Reply:
You are right

Reply:
Preserving baheviour